testnets/testnets.md

```diff
@@ -45,8 +45,9 @@ cd wasmd
 git checkout v0.10.0
 # generate coral executables
 make build-coral # make build-gaiaflex etc...
-
-# alias wasmd and cli to executable
+# add the executables to path
+export PATH="${PATH}:$(pwd)/build"
```

Comment:
nice addition.

do we still want the aliases below? I would just pick one testnet name and do it.
or do

```
NODE=corald
CLI=coral
```

and use $(NODE) and $(CLI) below instead of `wasmd`, `coral`, etc

Reply:
The latter looks better if we want to provide copy-paste-run script.

Reply:
If you alias `wasmd` to `.../corald` I guarantee you someone will end up running `wasmd` unaliased. How many times have you skimmed to the section you wanted and skipped the rest?

Reply:
But feel free to merge, if you are ready to answer questions

Reply:
You are right... Let's avoid aliasing and give a huge **NOTE** in the beginning of the document